internal/pkg/cli/cli.go

```diff
@@ -181,3 +184,40 @@ type storeReader interface {
 	archer.ApplicationLister
 	archer.ApplicationGetter
 }
+
+// returns true if error type is stack set not exist.
+func stackSetNotExistErr(err error) bool {
+	for {
+		if err == nil {
+			return false
+		}
+		aerr, ok := err.(awserr.Error)
+		if !ok {
+			return stackSetNotExistErr(errors.Unwrap(err))
+		}
+		if aerr.Code() != "StackSetNotFoundException" {
+			return stackSetNotExistErr(errors.Unwrap(err))
+		}
+		return true
+	}
+}
+
+// returns true if error type is stack not exist.
+func stackNotExistErr(err error) bool {
+	for {
+		if err == nil {
+			return false
+		}
+		aerr, ok := err.(awserr.Error)
+		if !ok {
+			return stackNotExistErr(errors.Unwrap(err))
+		}
+		if aerr.Code() != "ValidationError" {
+			return stackNotExistErr(errors.Unwrap(err))
+		}
+		if !strings.Contains(aerr.Message(), "does not exist") {
+			return stackNotExistErr(errors.Unwrap(err))
+		}
+		return true
+	}
+}
```

Comment:
I think the logic for these would be greatly simplified by moving them into the respective lower level packages since the errors wouldn't be wrapped by client code.

Reply:
Also will address the cloudformation package refactoring in the next PR including adding the missing app delete unit test.